src/connect/context/WagmiContext.tsx

```diff
@@ -0,0 +1,16 @@
+import { wagmiConfig } from '@/services';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactNode } from 'react';
+import { WagmiProvider } from 'wagmi';
+
+const queryClient = new QueryClient();
+
+export function WagmiContext({ children }: { children: ReactNode }) {
```

Comment:
sure can do

Reply:
@acaldas I've updated to pass in configs so that all the props can be customized. I left in default params for ease of use in mocks and stories. Let me know if this is good with you 👍